packages/core/src/oauth/twitch.ts

```
@@ -0,0 +1,41 @@
+import { getEnv } from "@/env.ts"
+import type { OAuthProviderCredentials } from "@/@types/index.ts"
+
+/**
+ * @see [Twitch - Get Users](https://dev.twitch.tv/docs/api/reference#get-users)
+ */
+export interface TwitchProfile {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)